Engine/Source/Runtime/Renderer/Private/PostProcess/PostProcessEyeAdaptation.cpp

```diff
@@ -677,9 +670,6 @@ void FSceneViewState::UpdatePreExposure(FViewInfo& View)
 			PreExposure = GetEyeAdaptationFixedExposure(View);
 		}
 	}
-	//AMCHANGE_begin: 
```

Comment:
Are you sure that nothing is changed by us in this code? How do you know that you have fully removed our changes?

Reply:
You're actually right, I did a diff with the file from epic's git repo and I removed a few AMChanged comments too much.
I added them back in.